bundles/org.openhab.binding.teslascope/README.md

```diff
@@ -0,0 +1,241 @@
+# Teslascope Binding
+
+This binding integrates [Tesla Electrical Vehicles](https://www.tesla.com).
+
+The integration happens through the [Teslascope](https://www.teslascope.com) API.
+
+## Supported Things
+
+All Tesla vehicles supported by Teslascope are supported..
+
+## Discovery
+
+Auto-discovery is not currently supported.
+
+## Thing Configuration
+
+As a minimum, the IP address is needed:
+
+- `apiKey` - apiKey provided by Teslascope
+- `publicID` - Vehicle Public ID listed in Teslascope
+- `refreshInterval` - The frequency with which to refresh information from Teslascope specified in seconds. Defaults to 60 seconds.
+
+## Channels
+
+All vehicles support a huge number of channels - the following list shows the standard ones:
+
+| Channel ID       | Item Type            | Label              | Description                                                                                 |
+|------------------|----------------------|--------------------|---------------------------------------------------------------------------------------------|
+| autoconditioning | Switch               | Auto Conditioning  | Turns on auto-conditioning (a/c or heating)                                                 |
+| batterylevel     | Number:Dimensionless | Battery Level      | State of the battery in %                                                                   |
```

Comment:
Looking at this table with channel id's, i would liek to ask you to do the task of aligning them with the official naming conventions: `lower-case-hyphen`
For reference : https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Reply:
embarassed to admin that I renamed all these already but forgot to update the readme.